components/drawer-layout/DrawerMenuContent.tsx

```
@@ -0,0 +1,99 @@
+import { Pressable, StyleSheet, Text, View } from "react-native";
+import { DrawerContentScrollView } from "@react-navigation/drawer";
+import { usePathname, useRouter } from "expo-router";
+import { APP_NAME } from "@/constants/App";
+import Colors from "@/constants/Colors";
+import { useTheme } from "@react-navigation/native";
+
+interface DrawerMenuContentProps { }
```

Biome

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)